client/recoverSecret.rs

```diff
@@ -0,0 +1,136 @@
+use std::ptr::null;
+use random_string::generate;
+
+pub struct RecoverSecretInput {
+    pub word_count: usize,
```

Comment:
struct cannot be created in another file for the comprehension ?

Reply:
Yes done